portal/static/js/src/modules/Consent.js

```diff
@@ -125,6 +121,23 @@ export default { /*global i18next datepicker $*/
             });
         });
     },
+    validateConsentDateFields: function(d, h, m, s) {
+        var errorMessage = "";
```

Comment:
I might use an array/list so you can decide how it's rendered at the end (eg concatenate for HTML, keep raw for JSON)